diff --git a/node_modules/eleventy-plugin-backlinks/lib/get-backlinks.js b/node_modules/eleventy-plugin-backlinks/lib/get-backlinks.js
index 87b0c18..3a9a814 100644
--- a/node_modules/eleventy-plugin-backlinks/lib/get-backlinks.js
+++ b/node_modules/eleventy-plugin-backlinks/lib/get-backlinks.js
@@ -6,7 +6,7 @@ function caselessCompare(a, b) {
 const wikilinkRegExp = /\[\[\s*([^\[\]\|\n\r]+)(\|[^\[\]\|\n\r]+)?\s*\]\]/g
 
 module.exports = function (options) {
-    return ({ collections, page }) => {
+    return async ({ collections, page }) => {
         const { notes } = collections;
 
         const fileStem = page.filePathStem.replace(`/${options.folder}/`, '');
@@ -15,7 +15,7 @@ module.exports = function (options) {
 
         // Search the other notes for backlinks
         for (const otherNote of notes) {
-            const noteContent = otherNote.template.frontMatter.content;
+            const noteContent = (await otherNote.template.read()).content;
 
             // Get all links from otherNote
             const outboundLinks = (noteContent.match(wikilinkRegExp) || [])
